src/content/docs/new-relic-only/basic-style-guide/writing-guidelines/code-formatting-guidelines-var-mark.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Code formatting guidelines: var and mark'
+contentType: page
+template: basicDoc
+topics:
+  - New Relic only
+  - Basic style guide
+  - Writing guidelines
+japaneseVersion: ''
+---
+
+This document serves as a supplement to the basic [`<var>`](/docs/new-relic-only/basic-style-guide/writing-guidelines/code-examples#annotate-var) and [`<mark>`](/docs/new-relic-only/basic-style-guide/writing-guidelines/code-examples#annotate-mark) documentation. It gives a longer explanation of when to use the `<var>` and `<mark>` formatting styles, and reasons for not using them.
+
+## var: Highlight user-specific values in procedural code snippets
+
+The `<var>` tag is used for user-input values in code snippets that customers would be using in procedural/functional ways. For example, you would use the `<var>` tag when explaining how to do some specific task.
+
+A `<var>` tag would not be used for:
+
+* Example code: If the code is meant to just be an example, to show the format/structure of the code, and there is not an actual procedure the customer is following, var tags are not needed. Examples:
+  * [Example configuration file](/docs/integrations/host-integrations/host-integrations-list/oracledb-monitoring-integration#example-config) (or this [Java agent config template](/docs/agents/java-agent/configuration/java-agent-config-file-template)): if you are showing an example config file that isn't part of a procedure, you shouldn't use the &lt;var> tag. Several reasons for this: 1) The important part of showing an example config file is to show the overall structure of the file, 2) Usually the number of config options present in a file will vary based on whatever the customer wishes to use, so using &lt;var> tags can actually be confusing as it implies that these values must be present.
```

Comment:
Looks like our codemods may be doing something weird with `<var>` and `<mark>` tags.

Reply:
I think this is intentional. Those tags are part of the text, so by using the html entity, it prevents it from rendering as a JSX component and instead outputs the literal text to the screen.